### ISSUE TITLE
Fade out always goes to 0

### ISSUE THREAD

**Issue by Niels-IO** [stefanwittwer/remotion-animated#6]
Hi @stefanwittwer,

Thanks for this awesome library. I'm having trouble to make a text fade to a value between 0 and 1. 

```javascript
<Animated
animations={[
Fade({to: 1, initial: 0}),
Scale({by: 1, initial: 0.3, overshootClamping: false}),
Fade({to: 0.8, start: 20}),
]}
style={{
transformOrigin: 'top center',
}}
>
<span> Title text</span>
</Animated>
```

This code will make the title text disappear and not clamp to the value of 0.8. Am I using it correctly, or is it a bug?

**Comment by stefanwittwer** [comment]
Hi there!
The opacity of the text in this example is indeed supposed animate from 1 to 0.8 in the second animation.
Let me look into it 👍

**Comment by tpetry** [comment]
I am having the same problem. I've also encountered that a fade from `0.0` to `1.0` never stops at `1.0`. It increases forever  - even after the defined `duration`.

**Comment by mecirmartin** [comment]
@stefanwittwer I have same problem, any update on this?

**Comment by mecirmartin** [comment]
@stefanwittwer I've dug into this and the problem is that in `interpolate` function, values are not clamped -> they leak outside the range when the frame is outside this range:
https://github.com/stefanwittwer/remotion-animated/blob/1d89dc3a57338aedb4c2a88cee640c925a0a9c19/packages/remotion-animated/src/animations/Fade.ts#L41C10-L41C15

I've created PR here:
https://github.com/stefanwittwer/remotion-animated/pull/10

**Comment by stefanwittwer** [comment]
@mecirmartin Thank you for digging into this and for taking the time to contribute a fix! The change is now merged and the fix is now published in version 2.1.1.
